gridt/src/app/movements/add-movement/add-movement.page.ts

```diff
@@ -80,17 +75,17 @@ export class AddMovementPage implements OnInit, OnDestroy {
 
     await el.present();
 
-    this.createMovementSubscription = this.api.createMovement$(this.form.value as Movement).subscribe(
+    this.subscriptions.push(this.api.createMovement$(this.form.value as Movement).subscribe(
       (message) => {
         el.dismiss();
         this.showMessage(message);
-        this.getMovementSubscription = this.api.getMovement$(this.form.value.name).subscribe();
+        this.subscriptions.push(this.api.getMovement$(this.form.value.name).subscribe());
```

Comment:
This will be rewritten in the future, allowing for a much more elegant way to do this.